src/utils/staking.js

```diff
@@ -228,7 +234,8 @@ export class Staking {
             await (await new nearApiJs.Account(this.wallet.connection, contractId)).state()
             return await new nearApiJs.Contract(this.wallet.getAccount(), contractId, { ...methods })
         } catch(e) {
-            throw new WalletError('No lockup contract for account', 'staking.errors.noLockup')
+            console.warn('No lockup contract for account')
+            // throw new WalletError('No lockup contract for account', 'staking.errors.noLockup')
```

Comment:
is this change desired? Best not to commit commented out code without explanation

Reply:
removed change